test/samples/typescript/expected.json

```diff
@@ -35,8 +35,8 @@
 				"messageId": "suggestUnknown",
 				"fix": {
 					"range": [
-						29,
-						32
+						58,
```

Comment:
These are the ranges of the suggestions. They are not yet mapped back correctly, so I had to change them here. Suggestions did not work before and do not now, so no change in functionality there.

Reply:
Does this imply that `--fix` will make bad edits to `.svelte` files? If so, shouldn't there be something in the README warning about this? I appreciate (as you said) that this isn't something new with this PR.

Reply:
No, this is only related to suggestions. Suggestions are different than fixes. You can select one of multiple suggestions in the IDE. That feature is not supported yet, independently of TS support.

Reply:
OK - thanks for clarifying :+1: